ddtrace/contrib/internal/botocore/services/bedrock.py

```diff
@@ -290,22 +306,23 @@ def _extract_streamed_response_metadata(
     elif provider == _STABILITY:
         # TODO: figure out extraction for image-based models
         pass
+
+    input_tokens = metadata.get("inputTokenCount", None)
+    output_tokens = metadata.get("outputTokenCount", None)
+    _set_llmobs_usage_from_invoke(ctx, input_tokens, output_tokens)
     return {
         "response.duration": metadata.get("invocationLatency", None),
-        "usage.prompt_tokens": metadata.get("inputTokenCount", None),
-        "usage.completion_tokens": metadata.get("outputTokenCount", None),
+        "usage.prompt_tokens": input_tokens,
+        "usage.completion_tokens": output_tokens,
     }
 
 
 def handle_bedrock_request(ctx: core.ExecutionContext) -> None:
     """Perform request param extraction and tagging."""
     request_params = _extract_request_params(ctx["params"], ctx["model_provider"])
     core.dispatch("botocore.patched_bedrock_api_call.started", [ctx, request_params])
-    prompt = None
-    for k, v in request_params.items():
-        if k == "prompt" and ctx["bedrock_integration"].is_pc_sampled_llmobs(ctx.span):
-            prompt = v
-    ctx.set_item("prompt", prompt)
+    if ctx["bedrock_integration"].is_pc_sampled_llmobs(ctx.span):
```

Comment:
```suggestion
    if ctx["bedrock_integration"].llmobs_enabled:
```
This shouldn't be dependent on sampling